src/compat/e-132/ArkoudaRandomCompat.chpl

```diff
@@ -2,14 +2,24 @@ module ArkoudaRandomCompat {
   use Random.PCGRandom only PCGRandomStream;
   record randomStream {
     type eltType = int;
-    forwarding var r: owned PCGRandomStream(eltType);
+    forwarding var r: shared PCGRandomStream(eltType);
     proc init(type t) {
       eltType = t;
-      r = new owned PCGRandomStream(eltType);
+      r = new shared PCGRandomStream(eltType);
     }
     proc init(type t, seed) {
       eltType = t;
-      r = new owned PCGRandomStream(eltType, seed);
+      r = new shared PCGRandomStream(eltType, seed);
+    }
+    proc ref fill(ref arr: []) where arr.isRectangular() {
+      var cpyArr = arr;
+      r.fillRandom(cpyArr);
+      arr = cpyArr;
```

Comment:
I'm not seeing why the copy is needed here?

Reply:
Thank you for catching this! I think I must've been misinterpreting an error I was hitting (and solved it some other way)

In my head I thought since `fillRandom` took and array literal back in 1.31 and `fill` takes a ref that this was necessary